dashboard/app/controllers/projects_controller.rb

```diff
@@ -67,7 +67,7 @@ def load
 
   def create_new
     return if redirect_under_13(@level)
-    redirect_to action: 'edit', channel_id: create_channel({
+    redirect_to action: 'edit', channel_id: ChannelToken.create_channel(request.ip, {
       name: 'Untitled Project',
       useFirebase: use_firebase_for_new_project?,
```

Comment:
I think this ref needs to be updated as well? `@level.game.use_firebase_for_new_project?`